tiny_cnn/core/conv_params.h

```diff
@@ -40,6 +40,13 @@ struct conv_layer_worker_specific_storage {
     vec_t prev_delta_padded_;
 };
 
+struct deconv_layer_worker_specific_storage {
+    const vec_t* prev_out_;
+    vec_t* curr_out_unpadded_;
+    vec_t curr_out_buf_;
+    vec_t curr_delta_padded;
+};
+
```

Comment:
@edgarriba According to the backend you have contributed, maybe I should get some suggestion from you to better adapt my codes with the backend using. I want to use a standalone worker storage here for deconvolutional padded delta and unpadded output.